cli.md

```diff
@@ -41,6 +43,7 @@ URL of parameter file in JSON format
 ### `-l, --log-level LOGLEVEL`
 
 Minimum Log level. One of `OFF`, `ERROR`, `WARN`, `INFO` (default), `DEBUG`, `TRACE`).
+:warning: This specification overwrites all other specifications. For a fine-grained change, please use the parameter file.
```

Comment:
What is the `parameter file`? And what if both are provided? Does this mean that when the command line flag is set, that it will override all other mechanisms? Even if those other mechanisms specify a lower or higher threshold for a certain area of the application? See https://github.com/OCR-D/core/issues/173